poll.js

```diff
@@ -0,0 +1,150 @@
+'use strict'
+var imgs = ['bag.jpg',
```

Comment:
It's usually considered a little cleaner to format things like this: 

```javascript
var imgs = [
  'bag.jpg',
  'banana.jpg',
  //  ...
];
```

Also, can you think of a clearer name for this variable?